daq/faucet_state_collector.py

```diff
@@ -309,6 +327,19 @@ def process_port_state(self, timestamp, name, port, status):
 
             port_table[KEY_PORT_STATUS_COUNT] = port_table.setdefault(KEY_PORT_STATUS_COUNT, 0) + 1
 
+    @dump_states
+    def process_lag_state(self, timestamp, name, port, status):
+        """process lag change event"""
+        topo_state = self.topo_state
+        with self.lock:
+            egress_table = topo_state.setdefault(EGRESS_STATE, {})
+            if status or name == egress_table.get(EGRESS_STATUS):
+                egress_table[EGRESS_STATUS] = name if status else "DOWN"
+                egress_table[EGRESS_PORT] = port if status else None
+                egress_table[EGRESS_TS] = datetime.fromtimestamp(timestamp).isoformat()
+                egress_table[EGRESS_CHANGE_COUNT] =\
```

Comment:
don't know what happned to my other comment, but I still haven't seen the answer as to why you need to call setdefault for EGRESS_CHANGE_COUNT when the first thing you do is overwrite the variable.  You can just use .get()